cbits/ed25519/ed25519.c

```diff
@@ -11,6 +11,7 @@
 #include "ed25519.h"
 #include "ed25519-randombytes.h"
 #include "ed25519-hash.h"
+#include "ed25519-cryptonite-exts.h"
```

Comment:
I think that's great to distinguish the exts parts. I did some extensions in another project, and should really have use this kind of idea !

Reply:
Almost everything is in header files, so I'm just continuing this trend and working around static declarations...